x-pack/plugins/security_solution/cypress/screens/alerts.ts

```diff
@@ -70,8 +69,7 @@ export const ALERTS_HISTOGRAM_PANEL_LOADER = '[data-test-subj="loadingPanelAlert
 
 export const ALERTS_CONTAINER_LOADING_BAR = '[data-test-subj="events-container-loading-true"]';
 
-export const NUMBER_OF_ALERTS =
-  '[data-test-subj="events-viewer-panel"] [data-test-subj="server-side-event-count"]';
+export const NUMBER_OF_ALERTS = ALERTS_COUNT;
```

Comment:
If with the current changes the locator used for `NUMBER_OF_ALERTS` is not used anymore, instead of doing this, we should change `NUMBER_OF_ALERTS` for `ALERTS_COUNT` in all the tests where we are using it and delete it from here.